test/controllers/usersController.test.js

```diff
@@ -0,0 +1,85 @@
+const request = require('supertest');
+const app = require('../../app');
+const User = require('../../app/models').user;
+
+describe('POST #signup', () => {
+  test('With valid params, it creates a User', async done => {
+    const response = await request(app)
+      .post('/users')
+      .send({
+        firstName: 'Test',
+        lastName: 'McTesting',
+        email: 'test@wolox.com.ar',
+        password: '12345678'
+      });
+    expect(response.statusCode).toEqual(201);
+    expect(response.body).toHaveProperty('id');
```

Comment:
bring in the user model and actually check the db to see if it was saved correctly with the same parameters